app/templates/home.html

```diff
@@ -1,21 +1,34 @@
 {% extends "partials/index/index_base.html" %}
 
 {% block content %}
+<script src="https://unpkg.com/sweetalert/dist/sweetalert.min.js"></script>
+<script src="{{ url_for('static', path='/js/dates_calculator.js') }}"></script>
 
+<div class="container mt-4">
+</div>
```

Comment:
What is the purpose of this div?

Reply:
fixed

Reply:
It doesn't exist in the new commit

Reply:
I can still se it here. Is there a new PR or something?

Reply:
Every time I pull from the develop this div exists